hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemConfigurations.java

```diff
@@ -49,7 +49,11 @@
   public static final int DEFAULT_AZURE_LIST_MAX_RESULTS = 500;
 
   public static final int MAX_CONCURRENT_READ_THREADS = 12;
-  public static final int MAX_CONCURRENT_WRITE_THREADS = 8;
+  public static final int DEFAULT_WRITE_CONCURRENCY_FACTOR = 4;
```

Comment:
concurrency reduced to 4 ? why.

Reply:
Not changed, Removed an unused constant. This value 4 is currently hardcoded in AbfsOutputStream class. Now made it a config.